### PR TITLE
Ice dispenser adjustments

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Bumped up total amount of ice chunks to 10000 since you cant restock the ice dispenser, removed the 28 Y pixel shift, made the dispsenser not display the total amount of ice as well as buffed the cooling power of ice. Now, a ratio of 1 ice to 2 reagent will make the mix cold. Also added the ability to dispense ice directly into a cup

## Why's this needed? 
Polishing some minor oversights and making ice colder should make it more usable

```changelog
(u)Carbadox
(+)Ice dispensers now pour ice into containers if interacted with a glass in hand.
```

### CODE REVIEW

**Comment by vode-code** [comment]
You could make the ice dispenser actually be "hooked into the station water mains" and refill, which would also prevent people from taking out a giant amount of ice at once and just using it for matsci or whatever.

**Comment by Carbadox** [comment]
> 
> 
> You could make the ice dispenser actually be "hooked into the station water mains" and refill, which would also prevent people from taking out a giant amount of ice at once and just using it for matsci or whatever.

Cleaner and less effort to just make it be alot ¯\_(ツ)_/¯

**Comment by geneva-breaker** [comment]
> Cleaner and less effort to just make it be alot of ice ¯_(ツ)_/¯

Less effort perhaps. 
Vode is right though.

**Comment by Carbadox** [comment]
No ones gonna abuse ice. If they do then a fix can be made. Otherwise, I see no difference in making the amount of ice be alot vs making it replenish. There's no difference in functionality and it's honestly a pointless nit pick for a minor feature meant to enchance bartendering

**Comment by ZeWaka** [comment]
> No ones gonna abuse ice. 

you would be surprised. ice is just more useful water

File: code/obj/item_dispensers.dm [hunk 1]
```diff
@@ -118,3 +118,17 @@
 	amount = 10000
 	display_amount = 0
 	pixel_y = 0
+	flags = FPRINT | NOSPLASH
+
+	New()
+		..()
+		src.create_reagents(10)
```
Comment (Tarmunora) [inline code/obj/item_dispensers.dm:125]: ?

**Comment by Azrun** [comment]
Miners send ice to catering! Rockbox integration!

**Review by ZeWaka — CHANGES_REQUESTED** [review]
x

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**